.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,42 @@
+name: "Feature Request & Ideas 🚀"
+description: Suggest a new feature or improvement (this does not mean you have to implement it)
+title: "Feature Request: "
+labels:
+  - feature
+  - backlog
+assignees: "marvinbuss"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this report! Before submitting, please make sure to [search through existing issues](https://github.com/Azure/data-product-streaming/issues) and [check the Known Issues section](/docs/EnterpriseScaleAnalytics-KnownIssues.md).
+
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature or Idea - What?
+      placeholder: Briefly describe what the new feature, idea or enhancement entails.
+    validations:
+      required: true
+
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature or Idea - Why?
+      placeholder: Briefly describe why the new feature, idea or enhancement is required and what you are trying to achieve.
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: code_of_conduct
+    attributes:
+      label: Code of Conduct
+      description: The Code of Conduct helps create a safe space for everyone. We require that everyone agrees to it.
```

Comment:
Reword , please. 

"The Code of Conduct helps create a safe space for everyone. We require that everyone abide by it."